homeassistant/components/juicenet/sensor.py

```diff
@@ -117,22 +112,4 @@ def icon(self):
     @property
     def state(self):
         """Return the state."""
-        state = None
-        sensor_type = self.entity_description.key
-        if sensor_type == "status":
-            state = self.device.status
-        elif sensor_type == "temperature":
-            state = self.device.temperature
-        elif sensor_type == "voltage":
-            state = self.device.voltage
-        elif sensor_type == "amps":
-            state = self.device.amps
-        elif sensor_type == "watts":
-            state = self.device.watts
-        elif sensor_type == "charge_time":
-            state = self.device.charge_time
-        elif sensor_type == "energy_added":
-            state = self.device.energy_added
-        else:
-            state = "Unknown"
-        return state
+        return getattr(self.device, self.entity_description.key, "Unknown")
```

Comment:
It may be considered a breaking change but our standard is to return `None` for unknown state. This will be converted in the base Entity class to `STATE_UNKNOWN`.

Reply:
It was "Unknown" before, using `None` now is what may break it https://github.com/home-assistant/core/pull/54390#discussion_r685918072

Reply:
Well technically "Unknown" could not be gotten originally as it was all if/else if based on the key for that sensor and sensors are hardcoded. So if the value was not in the self.device dict then it would have been `None` returned...

Reply:
I have reverted the commit so its back to `None`

Reply:
How would `None`  be returned when there is an `else` setting `state` to `"Unknown"`?

Reply:
Because the if/else is based on the static set keys, not the data returned

Reply:
I don't get that, but that's fine :+1:

Reply:
Ah ok I get what you are saying now, but since sensors are hardcoded to a specific set, "Unknown" should have never been reachable in the past anyway, therefore not really a breaking change.